api/rls.proto

```diff
@@ -31,34 +38,59 @@ message RateLimitRequest {
 // A RateLimitDescriptor is a list of hierarchical entries that are used by the service to
 // determine the final rate limit key and overall allowed limit. Here are some examples of how
 // they might be used for the domain "envoy".
-// 1) ["authenticated": "false"], ["ip_address": "10.0.0.1"]
-//    What it does: Limits all unauthenticated traffic for the IP address 10.0.0.1. The
-//    configuration supplies a default limit for the ip_address field. If there is a desire to raise
-//    the limit for 10.0.0.1 or block it entirely it can be specified directly in the
-//    configuration.
-// 2) ["authenticated": "false"], ["path": "/foo/bar"]
-//    What it does: Limits all unauthenticated traffic globally for a specific path (or prefix if
-//    configured that way in the service).
-// 3) ["authenticated": "false"], ["path": "/foo/bar"], ["ip_address": "10.0.0.1"]
-//    What it does: Limits unauthenticated traffic to a specific path for a specific IP address.
-//    Like (1) we can raise/block specific IP addresses if we want with an override configuration.
-// 4) ["authenticated": "true"], ["client_id": "foo"]
-//    What it does: Limits all traffic for an authenticated client "foo"
-// 5) ["authenticated": "true"], ["client_id": "foo"], ["path": "/foo/bar"]
-//    What it does: Limits traffic to a specific path for an authenticated client "foo"
+//
+// .. code-block:: cpp
+//
+//   ["authenticated": "false"], ["ip_address": "10.0.0.1"]
```

Comment:
nit: everywhere else in envoy and lyft ratelimit docs we use `remote_address` now. Not sure if you want to consolidate here too.